src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi/iscsi.component.ts

```diff
@@ -1,5 +1,6 @@
 import { Component } from '@angular/core';
 
+import { CellTemplate } from '../../../shared/enum/cell-template.enum';
 import { CephShortVersionPipe } from '../../../shared/pipes/ceph-short-version.pipe';
 import { DimlessBinaryPipe } from '../../../shared/pipes/dimless-binary.pipe';
```

Comment:
Now that we are no longer using the `dimlessBinaryPipe` in this file, we can remove this import and the corresponding line on the class constructor (line 25).